lib/internal/util.js

```diff
@@ -173,10 +174,12 @@ function deprecate(fn, msg, code, useEmitSync) {
     // constructor.
     deprecated.prototype = fn.prototype;
   }
-
+  deprecated[deprecate.isDeprecated] = true;
   return deprecated;
 }
 
+deprecate.isDeprecated = SymbolFor('nodejs.util.deprecate.isDeprecated');
```

Comment:
Is there a particular reason this is a global symbol in the registry? Cross version compat if we vendor deprecated stuff e.g. from strearms?

Reply:
No, no specific reason. I noticed that other Symbols within the `util` object were formatted the same way, and I wanted to follow the trend, is there a better way?